lib/Conversion/TritonGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -1767,6 +1772,11 @@ struct InsertSliceAsyncV2OpConversion
       return -1 -
              op.getValue().dyn_cast<IntegerAttr>().getValue().getZExtValue();
     }
+    if (!isa<BlockArgument>(v) &&
```

Comment:
I'm still a bit suspectable about the conditions. What if (1) `v` is a pointer and passed down into a device function, or (2) we add something to `v` and then subtract something from it?

Reply:
(2) We don't support the math now. The paras to construct TMA should come from kernel arguments directly.